src/components/Experiences.jsx

```diff
@@ -1,7 +1,37 @@
-import React from "react";
+/* eslint-disable */
+import React, { useEffect, useState } from "react";
+import { GetExperiences, GetNextExperiences } from "../api/experiences";
 
 const Experiences = () => {
-  return <div>Interview Experiences</div>;
+  const [InterviewExperiences, setInterviewExperiences] = useState();
+  useEffect(async () => {
```

Comment:
dont make useEffect async. Causes prob when navigating to other pages after interview exp page. 

```
useEffect(() => {
    GetExperiences().then((res) => setInterviewExperiences(res));
  }, []);
  ```
  do this and send PR